lib/Middleware/Cookie/CookieJar.php

```diff
@@ -84,4 +87,19 @@ public function clearExpiredCookies(): void
         $this->clear();
         $this->setCookies(array_values($cookies));
     }
+
+    /**
+     * Create an unique identifier for the cookie. Two cookies with the same identifier
+     * may have different values.
+     */
+    private function getHash(Cookie $cookie): string
+    {
+        return sha1(sprintf(
+            '%s|%s|%s|%s',
+            $cookie->getName(),
+            $cookie->getAttribute('domain'),
+            $cookie->getAttribute('path'),
+            $cookie->getAttribute('http')
```

Comment:
I don't think that `http` attribute exists.

Maybe use some constants?
https://github.com/kriswallsmith/Buzz/blob/62422cbbd17efa757353116d42508fda8503c754/lib/Middleware/Cookie/Cookie.php#L11-L19

Reply:
Of course, Excellent idea. 

Thanks!